aws-amplify/docs
# Adding tests for docs react components

_Description of changes:_ 

- Added `@testing-library/react` to test react components. 
- Added tests for `Block`,`BlockSwicher`, and `Callout`

More info on testing-library/react: https://testing-library.com/docs/react-testing-library/intro/

To test:
Run `yarn test`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
